meson_options.txt

```diff
@@ -5,5 +5,6 @@
 option('cpp-httplib_openssl', type: 'feature', value: 'auto', description: 'Enable OpenSSL support')
 option('cpp-httplib_zlib',    type: 'feature', value: 'auto', description: 'Enable zlib support')
 option('cpp-httplib_brotli',  type: 'feature', value: 'auto', description: 'Enable Brotli support')
+option('cpp-httplib_apple_keychain', type: 'feature', value: false, description: 'Enable loading certs from the Keychain on Apple devices')
```

Comment:
This option is a `boolean`, but here you're using a `feature` which has three states (enabled, disabled, and auto). If you want to keep the option, please correct the type.

Reply:
Fixed it